docs/docsite/rst/community/collection_development_process.rst

```diff
@@ -237,7 +143,7 @@ Here are some examples:
       remote_src=True even if mode was not set as a parameter.  This failed on
       filesystems which do not have permission bits (https://github.com/ansible/ansible/issues/29444).
 
-You can find more example changelog fragments in the `changelog directory <https://github.com/ansible/ansible/tree/stable-2.12/changelogs/fragments>`_ for the 2.12 release.
+You can find more example changelog fragments in the `changelog directory <https://github.com/ansible-community/ansible-build-data/blob/main/5/CHANGELOG-v5.rst>`_ for the Ansible 5 release.
```

Comment:
This paragraph is about changelog **fragments** (small YAML files), not about the combined changelog RST. How about the following instead:
```suggestion
You can find more example changelog fragments in the `changelog directory <https://github.com/ansible-collections/community.general/tree/main/changelogs/fragments>`_ for the community.general development branch.
```
Maybe it's better to link to a specific commit though, since this directory will suddenly be empty after the stable-5 branching happened.